doc/nrf/known_issues.rst

```diff
@@ -91,7 +91,7 @@ Sending data before connecting to nRF Cloud
 Other issues
 ============
 
-.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-4-99-dev1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```

Comment:
LTE sensor gateway crash was fixed and should be removed from known issues after v1.4.0 🙂